newm-server/src/main/kotlin/io/newm/server/database/migration/V56__UserSongMarketplaceUpdates.kt

```diff
@@ -0,0 +1,25 @@
+package io.newm.server.database.migration
+
+import org.flywaydb.core.api.migration.BaseJavaMigration
+import org.flywaydb.core.api.migration.Context
+import org.jetbrains.exposed.sql.transactions.transaction
+
+@Suppress("unused")
+class V56__UserSongMarketplaceUpdates : BaseJavaMigration() {
+    override fun migrate(context: Context?) {
+        transaction {
+            execInBatch(
+                listOf(
+                    "CREATE INDEX IF NOT EXISTS users_created_at_index ON users(created_at)",
```

Comment:
Do each of these have a particular query they are designed to improve, or is this just a shotgun approach? I'm wondering if there's a log we can turn on so we log any queries that run slowly and target just those for indexing.

Reply:
Yes, each is targeted to improve one or more of the existing Marketplace queries. Example: we have a couple of joints involving `users.id `-> `songs.owner_id ` and  `songs.id` ->` marketplace_sales.songId`. Also, we have deletion and sorting based on `created_at`

Reply:
Also, with the current (almost empty)  size of our tables, I don't think that we would see much difference in any query performance.